github_checkpatch.py

```diff
@@ -482,6 +482,9 @@ def post_review(self, review_input):
         score, event, comments, review_comment = \
             self.create_github_review(review_input, commit.sha)
 
+        if not comments and review_comment == "LGTM.  No errors found by checkpatch.":
+            return
+
```

Comment:
This seems like a hack. Couldn't we just not post the message to begin with?

Reply:
And what if the message changes for some reason?

Reply:
Yes, it's a hack.  The message comes from elsewhere in this file - on some codepaths the message is appended to.  IMO this code is end-of-life, medium term we should port checkpatch to GitHub actions and remove all this.

Reply:
Wow. I didn't realize this was a completely separate repo. Yes, agree we should EOL this.